src/com/lftechnology/java/training/dipak/Test.java

```diff
@@ -0,0 +1,28 @@
+package com.lftechnology.java.training.dipak;
+
+import java.util.Scanner;
+
+public class Test {
+
+	public static void main(String[] args) {
+		Scanner scanner=new Scanner(System.in);
+		System.out.print("Enter your name with Salutation::");
+		String inputName=scanner.nextLine();
+		inputName=inputName.trim();
+		
+		String[] text=inputName.split(" ");
+		
+		System.out.println("Salutation::"+text[0]);

```

Comment:
We never use println in real application, make habit of using logger. There are many third party in java like log4j, slf4j. You should be aware of synchronous vs asynchronous logger. 
http://www.mkyong.com/tutorials/log4j-tutorial/
http://stackoverflow.com/questions/5950557/good-examples-using-java-util-logging